### DISCUSSION TITLE
2024新年快乐 Jan 3.2024

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#49] [General]
2024年 阴历龙年九火运，火性大爆发,, 东方利财， 喜色黄金， 利好高能量， 甲辰年太岁位东南，犯太岁属相龙狗牛兔。
金水相生，不宜水木，宜金石，避免锋利物品，南方不动土，家居红色禁忌

**Comment by Lewisrxliu** [comment]
曹德旺2024年给年轻人的建议：客观公正地面对现实、积极地寻找解决办法、勇敢地走向未来！不会比60年更差，60年没有饭吃，不要悲观，不要扫兴，振作起来
https://view.inews.qq.com/a/20240102V07DGB00
https://m.weibo.cn/status/NA0IoFAHs

**Comment by guttasingapore** [comment]
2024 is gonna be even more crazy - elonmusk

https://twitter.com/elonmusk/status/1741611664393244938?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1741611664393244938%7Ctwgr%5E793d5c169d94ab74555b99c37ec2d62895d9053a%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.livemint.com%2Fnews%2Fworld%2F2024-is-gonna-be-more-crazy-elon-musk-predicts-heres-why-11704178918498.html

**Comment by Lewisrxliu** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/aa829b01-d3e0-498f-b383-e9ebaa6d209b)

**Comment by guttasingapore** [comment]
未来5年能源转型的IEA展望 （2024.1.11）
https://iea.blob.core.windows.net/assets/3f7f2c25-5b6f-4f3c-a1c0-71085bac5383/Renewables_2023.pdf

**Comment by Lewisrxliu** [comment]
周处除三害，寓意很深刻
https://mp.weixin.qq.com/s/g1mA5n-dHn4BXeu5MRBOuA
